ananse/influence.py

```diff
@@ -456,6 +457,120 @@ def save_reg_network(self, filename, full_output=False):
                         d["TF_act_target"],
                     ]
                     nw.write("\t".join(str(v) for v in cols) + "\n")
+#def plot_TF_GRN(self, infile, outfile, GRN_wb = False, network_algorithm = 'neato', n_TFs = 20, cmap = 'viridis', full_output = False):
+    def plot_TF_GRN(self, infile, outfile, GRN_wb = False, network_algorithm = 'neato', n_TFs = 20, cmap = 'viridis', full_output = False, cutoff_val = 0.1):
+        """
+        Plot the top20 of differential TFs and their interactions with the highest interaction score into a GRN network image
+
+        Parameters
+        ----------
+        self: influence_object to extract the diff network self.G, #future optimization = read diffnetwork instead
+        infile: influence output file
+        outfile: output file location
+        GRN_wb: when true and a -full--output influence GRN is suplied, use weighted binding interaction score to vizualize
+                edges instead of the probability score
+        network_algorithm: pyviz cluster algorithm used for node placement, options include: neato, dot, fdp, twopi, sfdp, circo 
+        n_TFs: number of (significantly differential expressed) Tfs to vizualize
+        cmap: matplotlib colour pallet used
+        full_output: if the GRN is a full output GRN (usefull for ploting weighted binding instead of prob score)
+        cutoff_val: minimum value the prob/wb score needs to have to be included in the GRN, 0.1 seems like a good number.
```

Comment:
This argument is related to the argument in my last comment. I'd suggest giving this argument matching names, something like `edge_score` and `edge_min`

Reply:
Ill change this to, good point.